_data/beyond-sidebar.yml

```diff
@@ -571,6 +591,9 @@
             - title: Mark pickup process as picked up
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-pickup-process-mark-picked-up
               id: mark_pickup_process_as_picked_up
+            - title: Merge multiple packing slips
```

Comment:
As discussed, I think this should be "Combine multiple pickup packing slips" (as you said, to differentiate between packing slips for shipping and pickup?)